docs/data-guidelines/features.md

```diff
@@ -0,0 +1,55 @@
+# Data guidelines for when to add features
+
+This file contains specific guidelines on when to add features to BCD, and the various types of features tracked in BCD.
+
+## Glossary
+
+- Feature: some trackable feature supported in a browser or runtime
+- Subfeature: any feature whose support is directly dependent on a parent (for example, the property of an interface)
+- Top-level feature: a feature that lives at the top of a category (for example, an interface, a CSS property, etc.)
+- Behavioral feature: a feature that describes the behavior of the browser or runtime (for example, security requirements, sorting, worker support, etc.)
+- Parameter feature: a feature that describes the support for a return type, method parameter, property value, etc.
+
+## Adding features and subfeatures
+
+A JSON file should be created for every _top level feature_ that has support in any of the browsers or runtimes tracked in BCD. For Web APIs, the top level features are individual Web API interfaces.
+
+A possible structure for a Web API is shown below:
+
+```
+category                // category name (always `api` for API interfaces)
+  interface               // top level feature
+    properties              // subfeature, add if supported
+      property values         // parameter subfeature, add all of them if not all values were supported since property was introduced
+    methods                // subfeature, add if supported
+      parameters              // parameter subfeature, add all of them if not all parameters were supported since method was introduced
+        parameter options       // parameter subfeature, add all of them if not all parameter options were supported since property was introduced
+        parameter type          // parameter subfeature, add all of them if not all parameter types were supported since property was introduced
+    support in workers     // behavioral subfeature, add if compatibility is different (might also just be on specific methods)
+```
```

Comment:
Ca we use nested lists here, rather than a code block?

```suggestion
- _category_ ← category name (always `api` for API interfaces)
  - _interface_ ← top-level feature
    - _properties_ ← subfeature, add if supported
      - _property values_ ← parameter subfeature, add all of them if not all values were supported since property was introduced
    - _methods_ ← subfeature, add if supported
      - _parameters_ ← parameter subfeature, add all of them if not all parameters were supported since method was introduced
        - _parameter options_ ← parameter subfeature, add all of them if not all parameter options were supported since property was introduced
        - _parameter type_ ← parameter subfeature, add all of them if not all parameter types were supported since property was introduced
    - _worker support_ ← behavioral subfeature, add if compatibility is different (might also just be on specific methods)
```